app/components/mappingsTable/mappingsTable.component.spec.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+import testUtils from '../testUtils/testUtils.js';
+
+/* jasmine specs for controllers go here */
+describe('Concept dictionary controllers', function() {
+
+    beforeEach(function(){
+        jasmine.addMatchers({
+            toEqualData: function(util, customEqualityTesters) {
+                return {
+                    compare: function(actual, expected) {
+                        var passed = angular.equals(actual, expected);
+                        return {
+                            pass: passed,
+                            message: 'Expected ' + actual + (passed ? '' : ' not') + ' to equal ' + expected
+                        }
+                    }
+                }
+            }
+        });
+    });
+
+    beforeEach(angular.mock.module('conceptDictionaryApp'));
+
+    describe('component: mapppingsTable', function() {
+        var $componentController, component, scope, $httpBackend;
+
+
+        beforeEach(inject(function(_$httpBackend_, $rootScope, _$componentController_, openmrsRest) {
+
+            $httpBackend = _$httpBackend_;
+            $componentController = _$componentController_;
+            scope = $rootScope.$new();
+        }));
+
+        it('should fetch ConceptMappings, ConceptMapTypes and ConceptSources at initialization', function(){
+            $httpBackend.whenGET('manifest.webapp').respond(500, "");
+            $httpBackend.whenGET(/translation.*/).respond();
+            $httpBackend.whenGET('components/indexMenu/indexMenu.html').respond();
+
+            $httpBackend.whenGET('/ws/rest/v1/conceptmaptype').respond({results: []});
+            $httpBackend.whenGET('/ws/rest/v1/conceptsource').respond({results: []});
+            $httpBackend.whenGET('/ws/rest/v1/concept/uuid/mapping?v=full').respond({results: []});
+
+
+            component = $componentController('mappingsTable', {scope: scope}, {conceptUuid: "uuid"});
+            $httpBackend.flush();
+            expect(component.mapTypes).toEqualData([]);
+            //only one default source to search all
+            expect(component.sources.length).toEqualData(1);
+            expect(component.mappings).toEqualData([]);
+        });
+
+        describe('component: mappingsTable, test functions', function(){
+            var mockMapping1, mockMapping2, mockMapping3, mockMappings, notificationMock;
+            notificationMock = testUtils.getNotificationMock();
+
+            beforeEach(function(){
+                mockMapping1 = {conceptReferenceTerm: {conceptSource: {uuid: "uuid"}, code: "code"}, conceptMapType: {uuid: "uuid"}, uuid: "uuid11" };
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2741260426/issues/source?bid=3196969&fileBranchId=3354756#l59)